src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -280,6 +280,9 @@ void dcd_int_enable (uint8_t rhport)
   NVIC_EnableIRQ(USB_LP_CAN_RX0_IRQn);
   NVIC_EnableIRQ(USBWakeUp_IRQn);
 #elif CFG_TUSB_MCU == OPT_MCU_STM32F1
+  NVIC_SetPriority(USB_HP_CAN1_TX_IRQn, 15);
```

Comment:
the irq priority should be set by application, in this case, it is in bsp_init(). Setting it here is not flexible enough for all the application.